jmh-benchmarks/src/main/java/org/apache/kafka/jmh/assignor/TargetAssignmentBuilderBenchmark.java

```diff
@@ -186,9 +190,33 @@ private void createAssignmentSpec() {
                 Collections.emptyMap()
             ));
         }
-        assignmentSpec = new AssignmentSpec(members, HOMOGENEOUS);
+        groupSpec = new GroupSpecImpl(members, HOMOGENEOUS, Collections.emptyMap());
     }
 
+    public Map<Uuid, Map<Integer, String>> invertedTargetAssignment(
```

Comment:
is it not possible to unify this with ServerSideAssignorBenchmark#invertedTargetAssignment?

seems like some details are different but they have a lot in common

Reply:
I think it's fine, it's just going to be used in these two benchmarks and they have some differences

Reply:
I agree with Jeff. Let's try to reuse as they fundamentally do the same thing.

Reply:
nit: The method could be static.